modules/Forum/module.php

```diff
@@ -167,6 +167,13 @@ public function __construct(Language $language, Language $forum_language, Pages
         }
 
         ReactionContextsManager::getInstance()->provideContext(new ForumPostReactionContext($forum_language));
+
+        Notification::addType(
+            'forum_topic_reply',
+            $forum_language->get('forum', 'forum_topic_replies'),
+            Module::getIdFromName('Forum'),
+            ['alert' => false, 'email' => true],
```

Comment:
Alert should also be enabled by default thought like current version atleast since users must click follow topic themself